sssom/cli.py

```diff
@@ -628,7 +627,7 @@ def decorator(f):
 @main.command()
 @input_argument
 @output_option
-@dynamically_generate_sssom_options(MAPPING_SLOTS)
+@dynamically_generate_sssom_options(SSSOMSchemaView().mapping_slots)
```

Comment:
I wonder if this will defeat the purpose of the refactoring. I should have thought of this before. 

I am wondering now if we need to find a way to skip network access in SchemaView()?